### ISSUE TITLE
Incompatibility with Additional Additions

### ISSUE THREAD

**Issue by spoorn** [joaoh1/OkZoomer#64]
Latest versions of both mods for Fabric 1.18.2

Looks like a mixin conflict with AdditionalAdditions

```
[23:23:56] [Render thread/WARN]: @Redirect conflict. Skipping okzoomer.mixins.json:MouseMixin->@Redirect::replaceSpyglassMouseMovement(Lnet/minecraft/class_746;)Z from mod okzoomer with priority 1000, already redirected by AdditionalAdditions.mixins.json:MouseMixin->@Redirect::slowCamera(Lnet/minecraft/class_746;)Z from mod additionaladditions with priority 1000
```

[crash-2022-03-30_23.23.56-client.txt](https://github.com/EnnuiL/OkZoomer/files/8386504/crash-2022-03-30_23.23.56-client.txt)

**Comment by EnnuiL** [comment]
Of course the hacky thing has failed; Don't worry, I believe that I can replace my `@Redirect`s with the help of [MixinExtras](https://github.com/LlamaLad7/MixinExtras) so that this conflict won't happen, at least with my own mod

**Comment by MarioMastr** [comment]
theres also an incompatibility with the better third person mod

**Comment by EnnuiL** [comment]
> theres also an incompatibility with the better third person mod

That's a completely different issue affecting an area which Ok Zoomer already did its own part on preventing the issue from happening; See #62 for more
Ideally, both conflicting mods shouldn't use a Redirect at all _unless_ it's absolutely necessary, but here? I did use a Redirect, and I can solve part of the issue by removing my own Redirects

**Comment by MarioMastr** [comment]
> > theres also an incompatibility with the better third person mod
> 
> 
> 
> That's a completely different issue affecting an area which Ok Zoomer already did its own part on preventing the issue from happening; See #62 for more

Sorry I didn't realise an issue was already created for it (kinda)

**Comment by EnnuiL** [comment]
Fixed on Ok Zoomer 5.0.0-beta.6 and its removal of our `@Redirects`
